### PR TITLE
Update parser

### LINKED ISSUE #72: Change the command format to prevent any unintended actions
The format now uses - to parse the different fields. It may cause issues for people with hyphenated names.

### LINKED ISSUE #88: Update Parser to make it easier for user to type commands

### PR DESCRIPTION
**Pull request by mingyang143**
fixes #88 
fixes #72 
Changed:

- prefix in parser to make it easier for users to type commands
- hours is optional when adding a tutee or tutor

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddTuteeCommandParser.java [hunk 1]
```diff
@@ -38,21 +38,27 @@ public AddTuteeCommand parse(String args) throws ParseException {
...
+        }
+
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_HOURS);
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
-        Hours hours = ParserUtil.parseHours(argMultimap.getValue(PREFIX_HOURS).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
         Set<Subject> subjects = ParserUtil.parseSubjects(argMultimap.getAllValues(PREFIX_SUBJECT));
```
Comment (kohkakohla) [inline src/main/java/seedu/address/logic/parser/AddTuteeCommandParser.java:61]: Nice! Removed dead code. Well done!

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 2]
```diff
@@ -6,14 +6,14 @@
...
     /* Prefix definitions */
-    public static final Prefix PREFIX_NAME = new Prefix("-name ");
-    public static final Prefix PREFIX_PHONE = new Prefix("-handphone ");
-    public static final Prefix PREFIX_HOURS = new Prefix("-hours ");
-    public static final Prefix PREFIX_EMAIL = new Prefix("-email ");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("-address ");
-    public static final Prefix PREFIX_TAG = new Prefix("t/"); // TODO change?
-    public static final Prefix PREFIX_SUBJECT = new Prefix("-subject ");
-    public static final Prefix PREFIX_FILEPATH = new Prefix("-filepath ");
+    public static final Prefix PREFIX_NAME = new Prefix("\\n ");
```
Comment (estellelim) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java:9]: Please remember to enable the disabled tests!

File: src/main/java/seedu/address/logic/parser/AddTutorCommandParser.java [hunk 3]
```diff
@@ -38,16 +38,23 @@ public AddTutorCommand parse(String args) throws ParseException {
                 ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_PHONE,
                         PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_HOURS, PREFIX_TAG, PREFIX_SUBJECT);
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_HOURS)
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS)
                 || !argMultimap.getPreamble().isEmpty()) {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddTutorCommand.MESSAGE_USAGE));
         }
 
+        // Hours is optional for adding a tutee and will be set to 0 if unspecified
```
Comment (kohkakohla) [inline src/main/java/seedu/address/logic/parser/AddTutorCommandParser.java:46]: Good comments letting the reader know what the code does. Very readable, very demure!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -40,9 +42,33 @@ VolunTier is a **desktop app for managing contacts, optimized for use via a  Lin
...
+#### 1. **What is the CLI?**
+The CLI is a text-based interface where users interact with the operating system by typing commands. It allows for:
+- **Efficient operations** compared to GUI
+- **Advanced automation** with scripts and batch commands
+
+#### 2. **Opening the Terminal**
+
+#### On macOS:
+```shell
+Cmd + Space
```
Comment (kohkakohla) [inline docs/UserGuide.md:58]: Hey Mingyang, I don't think command + space is the default shortcut for opening the terminal. Maybe you might want to look into this further?

**Review by kohkakohla — CHANGES_REQUESTED** [review]
LGTM in general, but do fix the GFMD.

**Review by kohkakohla — APPROVED** [review]
LGTM! Well done, Mingyang! Excellent work as always. Cheers!

**Pull request merged**